docs/ANALYTICS.md

```diff
@@ -0,0 +1,24 @@
+# SDS Analytics
+
+These samples showcase some commonly performed data visualization of SDS data.
+
+## [Dataviews](https://github.com/osisoft/OSI-Samples-OCS/blob/main/docs/DATA_VIEWS_README.md)
+
+These samples highlight basic operations of Data Views for OCS, including creation, updating, getting data from and deletion of Data Views. Please see the [Dataviews](https://github.com/osisoft/OSI-Samples-OCS/blob/main/docs/DATA_VIEWS_README.md) readme for more information on this sample.
```

Comment:
grammatically, i think this needs to be either of these two phrasings:

- including creat**ing**, updating, getting data from and delet**ing** Data Views
- including creation **of**, updating, getting data from and deletion of Data Views

Reply:
I don't know why I'm looking at this for so long, but I also would have used the word "reading" instead of "getting data", and then put them in the 'CRUD' order:

- including creating, reading from, updating, and deleting Data Views